pylmod/client.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
I would concur, we can tear it apart after it is in hear, but since we are porting it to a standalone library it makes sense to me to have it here.  Then we can do PRs/commits of removing things or adding methods separately.  I do like the start of the API you have below too, so go ahead and keep that.